README.md

```diff
@@ -1,43 +1,24 @@
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
-
 ## Getting Started
 
-First, run the development server:
+### 1. Clone the repository
 
 ```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
+git https://github.com/appKom/onlinefondet.git
+cd onlinefondet
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
-
-This project uses [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to automatically optimize and load [Geist](https://vercel.com/font), a new font family for Vercel.
-
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js) - your feedback and contributions are welcome!
+### 2. Install dependencies
 
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/building-your-application/deploying) for more details.
+```bash
+npm install
+```
 
+or
 
-## ENV
+```bash
+yarn install
+```
 
-Id of portfolio to be used for performance and positions:
+### 3. Set up environment variables
 
-- SHAREVILLE_PORTFILIO_ID=""
+XXX
```

Comment:
Er dette riktig?

```suggestion
#### Create `.env` file:

```bash
cp .env.template .env
```

#### Fill inn environment variables:

- **NEXTAUTH_URL**: The URL where your app will run. When running locally, it will likely be http://localhost:3000/. In production, this would be your live domain.
- **NEXTAUTH_SECRET**: A secret key used to encrypt session tokens in NextAuth.js. You can generate one with the command: openssl rand -base64 32.
- **GOOGLE_CLIENT_ID**: The Client ID from the Google Developer Console. This is needed to enable Google OAuth authentication for your app.
- **GOOGLE_CLIENT_SECRET**: The Client Secret from the Google Developer Console. This is also needed to complete the Google OAuth authentication setup.
- **SUPABASE_URL**: The URL of your Supabase project. You can find this in the Supabase dashboard under Settings -> API.
- **SUPABASE_ANON_KEY**: The public anonymous key for interacting with the Supabase API. You can find this in the Supabase dashboard under Settings -> API.
- **SUPABASE_SERVICE_KEY**: The service role key for your Supabase project, which allows access to more sensitive operations like inserting or modifying data with elevated permissions.
```

Reply:
ser riktig ut :)